### DISCUSSION TITLE
Ar Object placement

### DISCUSSION THREAD

**Discussion by KaranGoyalByjus** [google/model-viewer#4007] [Q&A]
Hi,
Does model-viewer support continues tracking like we have in this example https://immersive-web.github.io/webxr-samples/hit-test.html

In example: - reticle is following camera movements and when you tap on screen gameObject appears at reticle location.

**Comment by elalish** [comment]
Yes, but we have a different UX. If you drag the object (finger inside the box that appears) it will move across the surfaces according to hit-test results. It will jump up to a higher plane automatically, but only drop down on release.
